Automattic/jetpack
# Add caching for invites

Fixes #

## Proposed changes:

* Cache user invite information to reduce requests to user invite endpoint.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Sync to Atomic site
* Navigate to `/wp-admin/users.php`
* Verify SSO changes are working.
* Add new users and verify the user's SSO Status is `Send Invite` status.
* Send multiple invites, verify the invites are sent out correctly, and you can accept the invite.
* Verify the user's SSO Status is `Connected` after they accept the site invitation.
